src/Serializer/RequestsSerializer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Serializer;
+
+use OpenEuropa\EPoetry\PropertyInfo\Extractor\MappingReflectionExtractor;
+use OpenEuropa\EPoetry\Type\CreateRequests;
+use Symfony\Component\Serializer\Encoder\YamlEncoder;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
+use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * Class RequestsSerializer.
```

Comment:
We should add a description of the scope of this class, what it does, etc.

Reply:
Fixed.